src/utils.ts

```diff
@@ -92,3 +95,42 @@ export function getPrescriptionTableEndpoint(
 ) {
   return `${fhirBaseUrl}/Encounter?_getpagesoffset=${pageOffset}&_count=${pageSize}&subject.name=${patientSearchTerm}&_revinclude=MedicationRequest:encounter&_revinclude:iterate=MedicationDispense:prescription&_has:MedicationRequest:encounter:status=${status}&_sort=-date&_tag=http%3A%2F%2Ffhir.openmrs.org%2Fext%2Fencounter-tag%7Cencounter`;
 }
+
+export function getMedicationsByConceptEndpoint(conceptUuid: string) {
+  return `${fhirBaseUrl}/Medication?code=${conceptUuid}`;
+}
+
+/**
+ * Given an array of CodeableConcept condings, return the first one without an associated system (which should be the uuid of the underyling concept)
+ * @param codings
+ */
```

Comment:
@ibacher  this is how it works currently, but thoughts on whether this is a standard that should be relied upon?

Reply:
Currently, this is the behaviour of the FHIR2 module, i.e., the only coding returned without a code system will be the UUID for the concept. This won't change without (at least) a major version of the FHIR2 module.

This is a little bit non-standard. So, let me explain a bit why we have this implementation (this is not germane to this PR, just describing general behaviour of the the FHIR2 module). In an ideal world, FHIR wants a code system for every code value (since without a system a code is ambiguous). As I see it, leaves us with three choices:

1. The current implementation (omit a system for codes that represent UUIDs; codes without systems are, as I understand it, implicitly understood to be implementation-specific, i.e., not really "interoperable" but potentially locally useful).
2. Create a dummy "OpenMRS Concept UUID" system and use that.
3. Create an implementation-specific system URL for, e.g., "Mirebalais Concept UUID" or "PIH Concept UUID".

Option 3 is the most semantically correct way to do things, but I don't see a good way to work out the right URL (basically, how do I know what implementation the current system is running?). 

Option 2 would violate FHIR's concept semantics because, for example, http://fhir.openmrs.org/cs/concept-uuid|1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA might have a different meaning inside PIH's dictionary than in does in, e.g., Uganda EMRs. The general problem here is concepts derived from CIEL that reuse the CIEL UUIDs but which have been customised in implementation-specific ways may no longer be semantically equivalent—and we have no way of catching.

So we ended up implementing option 1.